api/apps/api/src/modules/projects/blm/change-blm-range.command.ts

```diff
@@ -2,6 +2,7 @@ import { Command } from '@nestjs-architects/typed-cqrs';
 import { Either } from 'fp-ts/Either';
 import { ProjectBlm } from '@marxan-api/modules/blm';
 
+export const forbidden = Symbol(`unauthorized access`);
```

Comment:
`blm` submodule is (or should) be blind on ACL as well 🤔

Reply:
I moved the `forbiddenError ` to the ACL completely and refactor how it is handled in the project-service, so now BLM should not have anything to do with this error regarding ACL apart from the own service method.

Check now because it should be now blind on ACL.